documentation/docs/getting_started/hornet_apt_repository.md

```diff
@@ -49,6 +49,15 @@ sudo service hornet start
 
 ### Managing the Node
 
+#### Unattended Upgrades
+
+Keeping up with Hornet releases can be automated by using [unattended system upgrades](https://wiki.debian.org/UnattendedUpgrades). To enable these, run:
```

Comment:
Should it better be mentioned, that also all other system updates will automatically be enabled?
I would also make clear in the Heading, that this is optional :)